network/src/connectivity_manager/test.rs

```diff
@@ -1275,62 +783,56 @@ fn basic_update_discovered_peers() {
-        .into_iter()
-        .collect();
-    let peers_1_2: TrustedPeerSet = vec![(peer_id_a, peer_a_1_2), (peer_id_b, peer_b1)]
-        .into_iter()
-        .collect();
+    let pubkeys_map_1 = hashmap! {peer_id_a => pubkeys_1.clone()};
+    let pubkeys_map_1_2 = hashmap! {peer_id_a => pubkeys_1_2, peer_id_b => pubkeys_1};
+
+    let peers_1 = hashmap! {peer_id_a => peer_a1};
+    let peers_2 = hashmap! {peer_id_a => peer_a2, peer_id_b => peer_b1.clone()};
+    let peers_1_2 = hashmap! {peer_id_a => peer_a_1_2, peer_id_b => peer_b1};
 
     // basic one peer one discovery source
     conn_mgr.handle_update_discovered_peers(DiscoverySource::OnChain, peers_1.clone());
-    assert_eq!(&*trusted_peers.read(), &pubkeys_map_1);
+    assert_eq!(&*eligible.read(), &pubkeys_map_1);
 
     // same update does nothing
     conn_mgr.handle_update_discovered_peers(DiscoverySource::OnChain, peers_1.clone());
-    assert_eq!(&*trusted_peers.read(), &pubkeys_map_1);
+    assert_eq!(&*eligible.read(), &pubkeys_map_1);
 
     // reset
     conn_mgr.handle_update_discovered_peers(DiscoverySource::OnChain, TrustedPeerSet::new());
-    assert_eq!(&*trusted_peers.read(), &pubkeys_map_empty);
+    assert_eq!(&*eligible.read(), &pubkeys_map_empty);
 
     // basic union across multiple sources
     conn_mgr.handle_update_discovered_peers(DiscoverySource::OnChain, peers_1);
-    assert_eq!(&*trusted_peers.read(), &pubkeys_map_1);
+    assert_eq!(&*eligible.read(), &pubkeys_map_1);
     conn_mgr.handle_update_discovered_peers(DiscoverySource::Config, peers_2);
-    assert_eq!(&*trusted_peers.read(), &pubkeys_map_1_2);
+    assert_eq!(&*eligible.read(), &pubkeys_map_1_2);
 
     // does nothing even if another source has same set
     conn_mgr.handle_update_discovered_peers(DiscoverySource::OnChain, peers_1_2.clone());
-    assert_eq!(&*trusted_peers.read(), &pubkeys_map_1_2);
+    assert_eq!(&*eligible.read(), &pubkeys_map_1_2);
     conn_mgr.handle_update_discovered_peers(DiscoverySource::Config, peers_1_2);
-    assert_eq!(&*trusted_peers.read(), &pubkeys_map_1_2);
+    assert_eq!(&*eligible.read(), &pubkeys_map_1_2);
 
     // since on-chain and config now contain the same sets, clearing one should do nothing.
     conn_mgr.handle_update_discovered_peers(DiscoverySource::Config, TrustedPeerSet::new());
-    assert_eq!(&*trusted_peers.read(), &pubkeys_map_1_2);
+    assert_eq!(&*eligible.read(), &pubkeys_map_1_2);
 
     // reset
     conn_mgr.handle_update_discovered_peers(DiscoverySource::OnChain, TrustedPeerSet::new());
-    assert_eq!(&*trusted_peers.read(), &pubkeys_map_empty);
+    assert_eq!(&*eligible.read(), &pubkeys_map_empty);
 
     // empty update again does nothing
     conn_mgr.handle_update_discovered_peers(DiscoverySource::Config, TrustedPeerSet::new());
-    assert_eq!(&*trusted_peers.read(), &pubkeys_map_empty);
+    assert_eq!(&*eligible.read(), &pubkeys_map_empty);
```

Comment:
I really wan to rename `eligible` to `trusted_peers` everywhere.  Only in connectivity manager is it `eligible`, because it really doesn't make much sense outside of the context of this component

Reply:
ugh definitely agree. would you prefer to do it in this commit or do it in a separate pass?

Reply:
Let's do it in a separate pass, I'm adding trusted peers more places.  I want to clean up the typing around it everywhere at once